book/introduction.md

```diff
@@ -3,34 +3,33 @@
 
-If you are using InnerSource in your company already and want to contribute your experiences to this book we would love to [welcome your contributions](../book/contribute-to-this-book.md)!
+If you are using InnerSource in your company already and want to contribute your experiences to this book, we would love to [welcome your contributions](../book/contribute-to-this-book.md)!
 
 ## What is InnerSource?
 
 We define InnerSource as:
 
-> The use of open source principles and practices for software development within the confines of an organisation.
+> The use of open-source principles and practices for software development within the confines of an organisation.
 
-InnerSource takes the lessons learned from developing open source software and applies them to the way companies develop software internally. As developers have become accustomed to working on world class open source software, there is a strong desire to bring those practices back inside the firewall and apply them to software that companies may be reluctant to release.
+InnerSource takes the lessons learned from developing open-source software and applies them to the way companies develop software internally. As developers have become accustomed to working on world-class open-source software, there is a strong desire to bring those practices back inside the firewall and apply them to software that companies may be reluctant to release.
 
-For companies building mostly closed source software, InnerSource can be a great tool to help break down silos, encourage internal collaboration, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.
+For companies building mostly closed source software, InnerSource can be a great tool to help break down silos, encourage and scale internal collaboration, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open-source world.
 
 ## What are InnerSource Patterns?
 
-Patterns are a way of describing a repeatable, proven solution to a problem within a context. They follow a simple form that helps people wanting to implement the solution to understand the constraints on the problem, the forces that must be balanced and the resulting context - the situation created by applying the solution.
+Patterns are a way of describing a repeatable, proven solution to a problem within a context. Patterns follow a simple form that assists you during implementation of a solution to understand the constraints of the problem, understand the forces you need to balance, and the resulting context - the situation created by applying the solution.
 
 In inner sourcing, patterns can provide a way for the InnerSource Commons participants to concisely share information with each other, improving the practice of inner sourcing. Each of the patterns are divided into Title, Problem Statement, Context, Forces, and Solutions as their main sections.
```

Comment:
```suggestion
Patterns can provide a way for the InnerSource Commons participants to concisely share information with each other, improving the practice of InnerSource. Each of the patterns are divided into Title, Problem Statement, Context, Forces, and Solutions as their main sections.
```

We want to stick to the [InnerSource](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/innersource-spelling.md) spelling as much as we can. So as I caught this here, I am fixing it as well.